File: businesslib/src/main/java/com/wujia/businesslib/listener/DownloadListener.java
package com.wujia.businesslib.listener;

/**
 * author ：shenbingkai@163.com
 * date ：2019-03-30
 * description ：
 */
public interface DownloadListener {

    void onTaskStart();

    void onTaskProgress(int percent,long currentOffset, long totalLength);

    void onTaskComplete(int state,String path); //0成功
}

File: businesslib/src/main/java/com/wujia/businesslib/listener/DownloadListener.kt
package com.wujia.businesslib.listener

/**
 * author ：shenbingkai@163.com
 * date ：2019-03-30
 * description ：
 */
interface DownloadListener {

    fun onTaskStart()

    fun onTaskProgress(percent: Int, currentOffset: Long, totalLength: Long)

    fun onTaskComplete(state: Int, path: String)  //0成功
}
